models/DataObject/Concrete.php

```diff
@@ -75,16 +75,16 @@ class Concrete extends DataObject implements LazyLoadedFieldsInterface
     /**
      * @internal
      *
-     * @var string
+     * @var string|null
      */
-    protected $o_classId;
+    protected ?string $o_classId = null;
```

Comment:
this change requires migration for the implementations on minor version, which should add the property typehints here: https://github.com/pimcore/pimcore/blob/fa3a9090ebc3fc826067c818edccb9a8ec7a80eb/lib/DataObject/ClassBuilder/FieldDefinitionPropertiesBuilder.php#L34
https://github.com/pimcore/pimcore/blob/83142b53cfd5a791e3c6167470a66336fd03da22/lib/DataObject/ClassBuilder/FieldCollectionClassBuilder.php#L68
https://github.com/pimcore/pimcore/blob/83142b53cfd5a791e3c6167470a66336fd03da22/lib/DataObject/ClassBuilder/ObjectBrickClassBuilder.php#L72

Reply:
Can we even provide a migration for that? 
As soon as we add the type to either (Concrete or real implementation, e.g. `Pimcore\Model\DataObject\Car`), it will break. 
So I think the only option is to add this manually in the course of upgrading to Pimcore 11. 
Or we remove that property type again.

Reply:
agreed solution: remove types for `$class_id` and `$class_name`.

Reply:
Removed types for both properties, removed the type in the build as well.